config/discogs-formats.yml

```diff
@@ -0,0 +1,346 @@
+# carriers
+4-TrackCartridge:

```

Comment:
Delete generated file and provide instructions for how to create them. Needed to support discogs vocab.

Reply:
got an error starting the app when i removed these files 🤔 /usr/local/bundle/gems/psych-3.3.4/lib/psych.rb:582:in `initialize': No such file or directory @ rb_sysopen - /app/samvera/hyrax-webapp/config/discogs-genres.yml (Errno::ENOENT)

Reply:
this will be resolved when I move the decorator over to the questioning auth gem